crates/nu-parser/src/parser.rs

```diff
@@ -5078,91 +5058,6 @@ pub fn parse_expression(
                     spans[0],
                 )),
             ),
-            b"const" => (
-                parse_call(
-                    working_set,
-                    &spans[pos..],
-                    spans[0],
-                    expand_aliases_denylist,
-                    is_subexpression,
-                )
-                .0,
-                Some(ParseError::ConstInPipeline(
-                    String::from_utf8_lossy(match spans.len() {
-                        1 | 2 | 3 => b"value",
-                        _ => working_set.get_span_contents(spans[3]),
-                    })
-                    .to_string(),
-                    String::from_utf8_lossy(match spans.len() {
-                        1 => b"variable",
-                        _ => working_set.get_span_contents(spans[1]),
-                    })
-                    .to_string(),
-                    spans[0],
-                )),
-            ),
-            b"mut" => (
-                parse_call(
-                    working_set,
-                    &spans[pos..],
-                    spans[0],
-                    expand_aliases_denylist,
-                    is_subexpression,
-                )
-                .0,
-                Some(ParseError::MutInPipeline(
-                    String::from_utf8_lossy(match spans.len() {
-                        1 | 2 | 3 => b"value",
-                        _ => working_set.get_span_contents(spans[3]),
-                    })
-                    .to_string(),
-                    String::from_utf8_lossy(match spans.len() {
-                        1 => b"variable",
-                        _ => working_set.get_span_contents(spans[1]),
-                    })
-                    .to_string(),
-                    spans[0],
-                )),
-            ),
-            b"alias" => (
```

Comment:
Do we have to adjust this specific error?

Reply:
Oh, sorry for my careless.   But I see an oppounity to simplify these error variant, we can merge `LetInPipeline, ConstInPipeline, MutInPipeline` into one `AssignInPipeline`